### PR TITLE
Fixing the kms key policy permissions

### PR DESCRIPTION
**Pull request by gopal9590**
I ran into this issue where the node gets terminated as soon as launch (and figured we had global EBS encryption enabled) , after updating the kms policy as documented here still didn't fix the issue , I figured out the resource should be "*" and  "kms:ReEncrypt*" - this has fixed the issue and now node is ready ! I'm really glad to get this fixed , hope noone else runs into the same , so would like to propose this change to troubleshooting here -

Fixes # 

**Description**

**How was this change tested?**

*

**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/troubleshooting.md [hunk 1]
```diff
@@ -416,12 +416,12 @@ To correct the problem if it occurs, you can use the approach that AWS EBS uses,
         "Action": [
             "kms:Encrypt",
             "kms:Decrypt",
-            "kms:ReEncrypt",
+            "kms:ReEncrypt*",
```
Comment (jonathan-innis) [inline website/content/en/preview/troubleshooting.md:419]:
```suggestion
            "kms:ReEncrypt",
```

Is this `*` supposed to be here?

**Comment by jonathan-innis** [comment]
Thanks for the contribution @gopal9590!

File: website/content/en/preview/troubleshooting.md (hunk 1 above)
Comment (gopal9590) [reply website/content/en/preview/troubleshooting.md:419]: Yea @jonathan-innis - Figured it is needed there as well , as per the AWS guide :

https://docs.aws.amazon.com/en_us/autoscaling/ec2/userguide/key-policy-requirements-EBS-encryption.html

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Comment by DanielJuravski** [comment]
\kind/improvement

**Pull request merged**